### PR TITLE
changing how the galasa version is retrieved

### PR DESCRIPTION
**Pull request by Akyiaa**
Signed-off-by: Fiona Ampofo <64271621+Akyiaa@users.noreply.github.com>

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

File: releasePipeline/30-deploy-maven-galasa.sh [hunk 1]
```diff
@@ -72,14 +72,23 @@ mkdir -p temp
...
+    rc=$?; 
+    if [[ "${rc}" != "0" ]]; then 
+      error "Failed to get galasa version"
+      exit 1
+    fi
 
-    echo "You are doing a release for version ${version}"
-}
+    # Note: We take the 2nd line which has an "<a href" string on... hopefully it won't change...
+    galasa_version=$(cat temp/galasa-version.txt | grep "<a href" | head -2 | tail -1 | cut -f2 -d'"' | cut -f1 -d'/')
```
Comment (KirbyKatcher) [inline releasePipeline/30-deploy-maven-galasa.sh:87]: can we look at using grep such as `<a * href=\"[0-9]*\.[0-9]*\.[0-9]*\/\"` instead in order to as that would be a bit more flexible?

**Comment by galasa-team** [comment]
Build successful

**Review by KirbyKatcher — APPROVED** [review]

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**